### ISSUE TITLE
astronomy (maybe other books) - bullet style none support in preface

### ISSUE THREAD

**Issue by lwtchu** [openstax/cnx-recipes#4386]
![Screenshot 2022-01-19 164241](https://user-images.githubusercontent.com/11299125/150230264-222043dc-fdae-4d4d-85b5-d9fd88ad3565.png)
![Screenshot 2022-01-19 164222](https://user-images.githubusercontent.com/11299125/150230265-17929bfd-8c9a-496c-995c-2ebf8503abae.png)

Can we keep the same indentation and everything, just remove the bullet? 

Mark up: `<list id="fs-id1163975534127" list-type="bulleted" bullet-style="none">`
Content:  https://github.com/openstax/osbooks-astronomy
Location: Preface

SIDE QUESTION - do we have support for bullet style none currently? Or no? Can someone also show me where list styles live (book specific or across library?)

**Comment by wilkus27** [comment]
@lwtchu about your side question from the description about issue: lists are styled separately in each of the theme. Since there are a lots of different styles for lists (like lists in chapter outline, learning objectives, notes etc.) we have very specific selectors for components. For example, we are styling separately lists in regular pages, composite pages, composite chapters and there we specify lists that are direct children of page, direct children of `page > section`, `page > section > section` etc. It was neceserry, otherwise styles for lists were overwritten. We developed this way of styling during work on each next theme, so at this moment it's a little bit messy. Themes have different shapes and different approaches, so it should be cleaned in the future. 

For this particular problem I added component for unordered no bullet list to `page > section > section`.
